corehq/apps/users/static/users/js/web_users.js

```diff
@@ -5,6 +5,7 @@ hqDefine("users/js/web_users",[
     "hqwebapp/js/initial_page_data",
     'bootstrap', // for bootstrap modal
     'hqwebapp/js/components.ko',    // pagination widget
+    'hqwebapp/js/knockout_bindings.ko', // for staticChecked data binding in web_users.html
```

Comment:
Why is this reverted? Isn't the staticChecked binding deleted?

Reply:
bad comment. updating